recipe/meta.yaml

```diff
@@ -30,14 +28,14 @@ requirements:
     - datasets !=2.5.0
     - importlib-metadata
     - filelock
-    - huggingface_hub >=0.23.0,<1
+    - huggingface_hub >=0.23.2,<1
     - numpy >=1.17
     - packaging >=20.0
     - pyyaml >=5.1
     - regex !=2019.12.17
     - requests
     - safetensors >=0.4.1
-    - tokenizers >=0.19,<0.20
+    - tokenizers >=0.20,<0.21
     - tqdm >=4.27
   run_constrained:
```

Comment:
Can you add run_constrained based on https://github.com/huggingface/transformers/blob/main/setup.py#L351 and  https://github.com/huggingface/transformers/blob/main/src/transformers/dependency_versions_table.py ?

Reply:
Done